crates/bevy_render/src/camera/camera.rs

```diff
@@ -350,10 +350,9 @@ pub fn camera_system<T: CameraProjection + Component>(
         })
         .collect();
 
-    let mut added_cameras = vec![];
-    for entity in &queries.p1() {
-        added_cameras.push(entity);
-    }
+    #[allow(clippy::needless_collect)]
```

Comment:
Whats the deal with this needless collect? False positive? Seems necessary to me.

Reply:
`needless_collect` is known to cause quite a few false positives.